refs #47: Adds Dockerfile for apache version.

michaelze opened the pull request:
Sorry it took me so long but somehow the holidays, new years eve and starting 2017 came into the way :)!

This is my first proposition on a apache version (#47) of this Dockerfile. Let me know what you think of and feel free to make any suggestions!

Greetings

Michael

pierreozoux commented:
Thanks for your PR :)

It would be amazing to use templates, like in https://github.com/docker-library/owncloud

michaelze commented:
Like this?

pierreozoux commented on apache/php.ini:
```diff
@@ -1,3 +1,4 @@
 always_populate_raw_post_data=-1
+sendmail_path = /usr/sbin/ssmtp -t
```
This is not needed as you can configure piwik to use php smtp.

pierreozoux commented on fpm/php.ini:
```diff
@@ -0,0 +1,4 @@
+always_populate_raw_post_data=-1
+sendmail_path = /usr/sbin/ssmtp -t
```
same as above

pierreozoux commented on update.sh:
```diff
@@ -10,4 +15,12 @@ latest="$(
+		s/%%VERSION%%/'"$latest"'/;
+		s/%%CMD%%/'"${cmd[$variant]}"'/;
+	' "$variant/Dockerfile"
+done
```
Could you remove the last char here?

pierreozoux replied:
Maybe you are using a windows system? do you see this red char from GitHub interface? if you use unix system, and there is noline at the end, this shouldn't appear.

pierreozoux commented:
Could you address the little comments please?

Thanks a lot for your work and patience :)

pierreozoux commented:
Last thing, can you update generate-stackbrew-library.sh to look like this one: https://github.com/nextcloud/docker/blob/master/generate-stackbrew-library.sh

Then, everything will be automatized!

Thanks!

michaelze commented:
Ah sorry, seems that I cloned this repo before you deleted the sendmail_path config from php.ini.
What exactly do you mean by "Could you remove the last char here"?

michaelze commented:
I addressed your comments.
You are right I am on a windows system but my editor should take care of line endings. Anyway, I added an extra newline to the end of every file that was missing one and that seems to have fixed GitHub's complaints :)!

michaelze commented:
So this PR has been out for nearly 2 months now. I even had to rebase my commit two times in the meantime due to new piwik versions being released (3.0.2 through 3.0.3).
Any chance for this getting accepted soon?

pierreozoux commented:
Really sorry for the delay, I'm a bit overloaded lately.

Thanks for your amazing work and your patience!

Pierre

michaelze commented:
@pierreozoux No problem, thanks for merging this!
The apache version is already available on the hub! Awesome :)
